azkaban-common/src/main/java/azkaban/project/NodeBean.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package azkaban.project;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The node bean is used by the YAML loader to deserialize DAG nodes
+ */
+public class NodeBean {
```

Comment:
Those will come later. I don't foresee that to be an issue. The current change is just to be able to read YML flows.